src/main/java/seedu/account/logic/commands/CommandResult.java

```diff
@@ -0,0 +1,73 @@
+package seedu.account.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Objects;
+
+/**
+ * Represents the result of a command execution.
+ */
+public class CommandResult {
+
+    private final String feedbackToUser;
+
+    /** Help information should be shown to the user. */
+    private final boolean showHelp;
+
+    /** The application should exit. */
+    private final boolean exit;
+
+    /**
+     * Constructs a {@code CommandResult} with the specified fields.
+     */
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+        this.feedbackToUser = requireNonNull(feedbackToUser);
+        this.showHelp = showHelp;
+        this.exit = exit;
+    }
+
+    /**
+     * Constructs a {@code CommandResult} with the specified {@code feedbackToUser},
+     * and other fields set to their default value.
+     */
+    public CommandResult(String feedbackToUser) {
+        this(feedbackToUser, false, false);
+    }
+
+    public String getFeedbackToUser() {
+        return feedbackToUser;
+    }
+
+    public boolean isShowHelp() {
+        return showHelp;
+    }
+
+    public boolean isExit() {
+        return exit;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof seedu.account.logic.commands.CommandResult)) {
+            return false;
+        }
+
+        seedu.account.logic.commands.CommandResult otherCommandResult = (seedu.account
```

Comment:
Perhaps you were trying to repackage everything, but maybe the refactoring messed up?